gnd/src/main/java/com/google/android/gnd/rx/Loadable.java

```diff
@@ -58,17 +52,10 @@ private Loadable(LoadState state, @Nullable T data, Throwable error) {
   }
 
   public static <T> Loadable<T> error(Throwable t) {
+    Timber.e(t);
```

Comment:
This is a transformation method that will return a new instance. It's assumed that the error in the Loadable will be handled somewhere, so printing the error to the log here will result in duplicate outputs in the log.

Reply:
Done